src/ChaiVM/interpreter/executor.cpp

```diff
@@ -48,47 +48,63 @@ void Executor::star(Instruction ins) {
     DO_NEXT_INS()
 }
 void Executor::add(Instruction ins) {
-    regFile_.acc() += regFile_[ins.r1];
+    regFile_.acc() =
```

Comment:
`std::bit_cast` is not worth using here. `std::bit_cast` may do `memcpy` (check https://en.cppreference.com/w/cpp/numeric/bit_cast), which slow for executor handlers. Why did you make these changes?